### PR TITLE
Link a liq 2.0 poll

### PR DESCRIPTION
**Pull request by prose11**
### Attention (CC)

@0xElihu 

## Summary of changes

Added poll, links for parameters still need to be added, I'll be around to do so if we need that done this weekend, it sounded like that might be a Monday thing though.

### CODE REVIEW

File: governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding LINK-A to the Liquidations 2.0 Framework in the Maker Protocol with the parameters below:
+
+**Price Function**
+* [Price Change Multiplier](https://community-development.makerdao.com/en/learn/governance/param-auction-price-function): 0.99
+* [Price Change Interval](https://community-development.makerdao.com/en/learn/governance/param-auction-price-function): 90 seconds
+* [Auction Price Multiplier](https://community-development.makerdao.com/en/learn/governance/param-auction-price-multiplier): 1.3
+* [Breaker Price Tolerance](https://community-development.makerdao.com/en/learn/governance/param-breaker-price-tolerance): 0.5
```
Comment (0xElihu) [inline governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md:28]: Change to "Circuit Breaker Price Tolerance"?

File: governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+**Global Parameters**
+* [Global Liquidation Limit](https://community-development.makerdao.com/en/learn/governance/param-global-liquidation-limit): 100 million DAI
+
+
+Please review the following [Risk Evaluation](https://forum.makerdao.com/t/link-a-liquidations-2-0-parameters/7180) containing information about these proposed parameters to inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* LINK-A will be added to the Liquidations 2.0 System by a future executive vote as the Smart Contracts Team schedule allows.
```
Comment (0xElihu) [inline governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md:47]: Change to "Smart Contract Team's schedule".

File: governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+* [Global Liquidation Limit](https://community-development.makerdao.com/en/learn/governance/param-global-liquidation-limit): 100 million DAI
+
+
+Please review the following [Risk Evaluation](https://forum.makerdao.com/t/link-a-liquidations-2-0-parameters/7180) containing information about these proposed parameters to inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* LINK-A will be added to the Liquidations 2.0 System by a future executive vote as the Smart Contracts Team schedule allows. 
+* It is expected that this executive vote will take place within 30 days of this poll passing, absent external factors.
```
Comment (0xElihu) [inline governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md:48]: What could be the external factors in this case? Maybe let's delete that phrase entirely?

File: governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+---
+title: Add LINK-A to Liquidations 2.0 Framework - April 5, 2021
+summary: This poll proposes system parameters which could be used to initialize LINK-A with the Liquidations 2.0 Framework.
```
Comment (0xElihu) [inline governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md:3]: Change to: "This poll proposes system parameters to be used in initializing LINK-A with the Liquidations 2.0 Framework"

File: governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+---
+# Poll: Add LINK-A to Liquidations 2.0 Framework - April 5, 2021
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the MakerDAO mandated actors. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, April 5 at 16:00 UTC.
+
+**This is a binary vote. In a binary vote, you may vote for a single option. You should vote for the option which you prefer. Alternatively, if you would accept either of the two options and have no preference between them, you should vote 'Abstain'.**
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding LINK-A to the Liquidations 2.0 Framework in the Maker Protocol with the parameters below:
```
Comment (0xElihu) [inline governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md:22]: Change to:  "This poll allows the MakerDAO Community to signal their support or opposition to adding LINK-A to the Liquidations 2.0 Framework with the parameters below:"

File: governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+**Global Parameters**
+* [Global Liquidation Limit](https://community-development.makerdao.com/en/learn/governance/param-global-liquidation-limit): 100 million DAI
+
+
+Please review the following [Risk Evaluation](https://forum.makerdao.com/t/link-a-liquidations-2-0-parameters/7180) containing information about these proposed parameters to inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
```
Comment (0xElihu) [inline governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md:46]: Add a comma in "[...] the 'No' option, then..."

File: governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+* LINK-A will be added to the Liquidations 2.0 System by a future executive vote as the Smart Contracts Team schedule allows. 
+* It is expected that this executive vote will take place within 30 days of this poll passing, absent external factors.
+* If the executive vote passes, then these changes will become active in the Maker Protocol after the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay) has expired.
+
+**If the votes for the 'No' option equal or exceed the votes for the 'Yes' option then no further action will be taken at this time.**
```
Comment (0xElihu) [inline governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md:51]: Add a comma in "[...] the 'Yes' option, then..."

File: governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md (hunk 1 above)
Comment (LongForWisdom) [reply governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md:28]: Trying to keep them to 3 words or less. We might change them later.

File: governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md (hunk 2 above)
Comment (LongForWisdom) [reply governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md:47]: If we make a change here, we should do it to all the templates.

File: governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md (hunk 3 above)
Comment (LongForWisdom) [reply governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md:48]: No idea, that's the point of it, it's meant to cover unforeseen circumstances which may result in a delay.

File: governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md (hunk 7 above)
Comment (LongForWisdom) [reply governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md:51]: Should make this change on the template.

**Review by LongForWisdom — APPROVED** [review]

**Pull request merged**